mobile_app_connector/security/ir.model.access.csv

```diff
@@ -36,3 +36,4 @@ admin_access_mobile_app_tile_type,admin access_mobile_app_tile_type,model_mobile
 admin_access_mobile_app_tile_subtype,admin access_mobile_app_tile_subtype,model_mobile_app_tile_subtype,sales_team.group_sale_manager,1,1,0,0
 user_access_mobile_app_feedback,User access app feedback,model_mobile_app_feedback,base.group_portal,1,1,1,0
 admin_access_mobile_app_feedback,Admin access app feedback,model_mobile_app_feedback,base.group_system,1,1,1,1
+create_access_mobile_app_public_contact_us,User not logged can create crm claim,model_crm_claim,base.group_public,1,1,1,0
```

Comment:
If you use sudo() inside the method `mobile_contact_us`, I think it's not needed to add that access rule which would open maybe too much rights